.github/workflows/healthchecks_influxdb_ci.yml

```diff
@@ -0,0 +1,57 @@
+name: HealthChecks InfluxDB CI
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ master ]
+    paths:
+      - src/HealthChecks.InfluxDB/**
+      - test/HealthChecks.InfluxDB.Tests/**
+      - .github/workflows/healthchecks_influxdb_ci.yml
+      - Directory.Build.props
+      - Directory.Build.targets
+    tags-ignore:
+      - release-*
+      - preview-*
+      
+  pull_request:
+    branches: [ master ]
+    paths:
+      - src/HealthChecks.InfluxDB/**
+      - test/HealthChecks.InfluxDB.Tests/**
+      - .github/workflows/healthchecks_influxdb_ci.yml
+      - Directory.Build.props
+      - Directory.Build.targets
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: |
+            6.0.x
+            7.0.x
+      - name: Restore
+        run: dotnet restore ./src/HealthChecks.InfluxDB/HealthChecks.InfluxDB.csproj &&
+             dotnet restore ./test/HealthChecks.InfluxDB.Tests/HealthChecks.InfluxDB.Tests.csproj
+      - name: Check formatting
+        run: |
+          dotnet format --no-restore --verify-no-changes --severity warn ./src/HealthChecks.InfluxDB/HealthChecks.InfluxDB.csproj || (echo "Run 'dotnet format' to fix whitespace issues" && exit 1) &&
+          dotnet format --no-restore --verify-no-changes --severity warn ./test/HealthChecks.InfluxDB.Tests/HealthChecks.InfluxDB.Tests.csproj || (echo "Run 'dotnet format' to fix analyzers issues" && exit 1)
+      - name: Build
+        run: |
+          dotnet build --no-restore ./src/HealthChecks.InfluxDB/HealthChecks.InfluxDB.csproj &&
+          dotnet build --no-restore ./test/HealthChecks.InfluxDB.Tests/HealthChecks.InfluxDB.Tests.csproj
+      - name: Setup InfluxDB
+        uses: maikebing/influxdb-action@v4

```

Comment:
Ok. Then ping back when fix will be merged in official repo. I will review today and merge.